tests-scan

```diff
@@ -144,110 +143,50 @@ def is_internal_context(context):
-    if job.number == 0:
-        invoke = "../tests-invoke --revision {revision} --repo {github_base}"
-    else:
-        invoke = "../tests-invoke --pull-number {pull_number} --revision {revision} --repo {github_base}"
-    test_env = "TEST_OS={image}"
-    wrapper = "./s3-streamer --repo {github_base} --test-name {name}-{current} " \
-              "--github-context {github_context} --revision {revision}"
-
-    if job.base:
-        quoted = shlex.quote(job.base)
-        test_env += f" BASE_BRANCH={quoted}"
-        checkout += f" --rebase={quoted}"
-
-    if job.bots_ref:
-        # we are checking the external repo on a cockpit PR, so stay on the project's main branch
-        checkout += " {ref}"
-        test_env += " COCKPIT_BOTS_REF={bots_ref}"
-    else:
-        # we are testing the repo itself, checkout revision from the PR
-        checkout += " {ref} {revision}"
-
-    if scenario:
-        test_env += " TEST_SCENARIO={scenario}"
-
-    cmd = " ".join([checkout, "&& cd make-checkout-workdir &&", test_env, invoke])
-    return (wrapper + ' -- /bin/sh -c "' + cmd + '"').format(
-        priority=priority,
-        name=shlex.quote(job.name),
-        revision=shlex.quote(job.revision),
-        ref=shlex.quote(job.ref),
-        bots_ref=shlex.quote(job.bots_ref or ''),
-        image=shlex.quote(image),
-        scenario=(shlex.quote(scenario)),
-        github_context=job.github_context,
-        current=current,
-        pull_number=job.number,
-        repo=shlex.quote(job.repo),
-        github_base=shlex.quote(options.repo),
-    )
-
-
 def queue_test(job: Job, channel, options: argparse.Namespace) -> None:
-    command = tests_invoke(job, options)
+    priority = min(job.priority, distributed_queue.MAX_PRIORITY)
 
-    if command:
-        priority = min(job.priority, distributed_queue.MAX_PRIORITY)
+    slug_suffix = job.github_context.replace('/', '-').replace('@', '-')
+    slug = f"{job.name}-{job.revision[:8]}-{time.strftime('%Y%m%d-%H%M%S')}-{slug_suffix}"
+    (image, _, scenario) = job.context.partition("/")
 
-        slug_suffix = job.github_context.replace('/', '-').replace('@', '-')
-        slug = f"{job.name}-{job.revision[:8]}-{time.strftime('%Y%m%d-%H%M%S')}-{slug_suffix}"
-        (image, _, scenario) = job.context.partition("/")
+    env = {
+        "TEST_OS": image,
+    }
+    if scenario:
+        env["TEST_SCENARIO"] = scenario
+    if job.bots_ref:
+        env["COCKPIT_BOTS_REF"] = job.bots_ref
+    if job.base:
+        env["BASE_BRANCH"] = job.base
 
-        env = {
-            "TEST_OS": image,
-        }
-        if scenario:
-            env["TEST_SCENARIO"] = scenario
-        if job.bots_ref:
-            env["COCKPIT_BOTS_REF"] = job.bots_ref
-        if job.base:
-            env["BASE_BRANCH"] = job.base
-
-        body = {
-            # old-style command line jobs
-            "command": command,
-            "type": "test",
+    body = {
+        "job": {
```

Comment:
I wonder if you'd like to add a `"human"` field here, like in `issue-scan`.

Reply:
Is this ok for a follow-up? Running out of time today